scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -21,38 +21,53 @@ This quickstart assumes that you are familiar with Terra workspaces, the data mo
     - BigQuery Read Session User
 4. These tools expect re-blocked gVCF files as input.
 
-## 1. Import Data
-In order to load data into BigQuery without hitting daily load limits, we recommend you group your input files into samples sets and follow these steps for each sample set.
+## 1. Setup workspace with data model
+This workspace has two files you can use to initially populate the data model. Both files are in the `Files` section of the workspace. One named `samples_10.tsv` has 10 samples with already reblocked gvcfs. The other file `samples_3202.tsv` has 3202 samples only a few of which have their reblocked gvcf attributes filled in. 
 
-A sample set for the quickstart has already been created with 10 samples and paths to re-blocked gVCFs for each sample.  Run the two import workflows against this sample set by selecting "sample_set" as the root entity type ("Step 1") and `gvs_demo-10` for the data ("Step 2").  If you are creating your own sample set, note that the sample table should have a column for the re-blocked gVCFs (`hg38_reblocked_gvcf` or `reblocked_gvcf_path`) and their index files need to be in the same location.
+## 2. Reblock samples
+Run the GvsAoUReblockGvcf workflow on the samples
```

Comment:
Can this just be the WARP pipeline?  Feels off to have people running an AoU specific pipelines in our quickstart

Reply:
Good point. I intended this workspace to be the template for the Gvs AoU specific workspace. I'll change the name of the markdown file for this use.